bindings/node.js/kzg.cxx

```diff
@@ -260,17 +237,19 @@ Napi::Value VerifyKzgProof(const Napi::CallbackInfo& info) {
 
 // verifyBlobKzgProof: (blob: Blob, commitmentBytes: Bytes48, proofBytes: Bytes48, setupHandle: SetupHandle) => boolean;
 Napi::Value VerifyBlobKzgProof(const Napi::CallbackInfo& info) {
-  auto env = info.Env();
-
-  size_t argument_count = info.Length();
-  size_t expected_argument_count = 4;
-  if (argument_count != expected_argument_count) {
-    return throw_invalid_arguments_count(expected_argument_count, argument_count, env);
+  Napi::Env env = info.Env();
+  Blob *blob_bytes = reinterpret_cast<Blob *>(get_bytes(env, info[0], BYTES_PER_BLOB, "blob"));
+  if (blob_bytes == nullptr) {
```

Comment:
Why return `Undefined` and not `Null`?

This way the verification functions can return four different things: `Null`, `Undefined`, `true`, `false`.

Wouldn't it make sense to return `Null` in all those cases to keep things simpler?
(Same for all occurences above)

Reply:
Sure, updated.   I actually misread the comment when I was making some changes and intuitively and inadvertently switched them all to `Undefined` due the nature of `null` in JS but reverted and then made everything `Null`.  Sorry for the extra commit.

As a note `null` can pose challenges in JS.  In this instance they value gets buried because of the pending exception but figured it was worth mentioning.  Among other things `typeof null === 'object'` for no logical reason